content/en/docs/concepts/distributions.md

```diff
@@ -51,6 +51,18 @@ distribution if they wish to use components in the
 [Registry](../../../registry) that are not upstreamed to the OpenTelemetry
 project.
 
+## Contribution or distribution?
+
+Before you read on and learn how you can create your own distribution, ask yourself if your
+additions on top of an OpenTelemetry component would be beneficial for everyone and therefore
+should be included in the reference implementations:
+
+- Can your scripts for "ease of use" be generalized?
+- Can your changes to default settings be the better option for everyone?
+- Are your additional packaging options really specific?
+- Might your test, performance & security coverage work with the reference implementation as well?
+- Have you checked with the community if your additional capabilities could be part of the standard?
+
```

Comment:
Is the goal here to make a difference between a contribution and distribution? The distinction feels implicit in the content, I would make it a little more explicit.

Reply:
The goal is that people reconsider if all the features they create for their distribution are unique to them or if they should give them back to the community.

How would you make it more explicit?